### PR TITLE
Changed style

### PR DESCRIPTION
**Pull request by TomyLobo**

### CODE REVIEW

File: generate-puppetfile [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
-sed -i.bak "s/^\/.*/forge 'http:\/\/forge.puppetlabs.com'\n/" $LIST
-sed -i.bak "s/-/\//" $LIST
-sed -i.bak "s/├── /mod '/" $LIST
-sed -i.bak "s/└── /mod '/" $LIST
-sed -i.bak "s/ (/', '/" $LIST
-sed -i.bak "s/v\([[:digit:]]\)/1/" $LIST
-sed -i.bak "s/)/'/" $LIST
+sed -i.bak "
+	s/^\/.*/forge 'http:\/\/forge.puppetlabs.com'\n/
+	s/-/\//" "$LIST"
```
Comment (rnelson0) [inline generate-puppetfile]: There's an errant `" "$LIST"` here.

File: generate-puppetfile [hunk 2]
```diff
@@ -1,29 +1,31 @@
 #!/bin/sh
-MODULEPATH=`mktemp -d`
-LIST=`mktemp`
-for module in $*
-do
+
+MODULEPATH="$(mktemp -d)"
+LIST="$mktemp"
```
Comment (rnelson0) [inline generate-puppetfile]: This should be `LIST="$(mktemp)"`

**Comment by rnelson0** [comment]
@TomyLobo Thanks for this patch! This needs two fixes before I can merge it, though, when you have the time.

File: generate-puppetfile (hunk 1 above)
Comment (TomyLobo) [inline generate-puppetfile]: whoopsie, fixing

File: generate-puppetfile [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
 
-puppet module list --modulepath=$MODULEPATH > $LIST
+puppet module list --modulepath="$MODULEPATH" > "$LIST"
 
-sed -i.bak "s/^\/.*/forge 'http:\/\/forge.puppetlabs.com'\n/" $LIST
-sed -i.bak "s/-/\//" $LIST
-sed -i.bak "s/├── /mod '/" $LIST
-sed -i.bak "s/└── /mod '/" $LIST
-sed -i.bak "s/ (/', '/" $LIST
-sed -i.bak "s/v\([[:digit:]]\)/1/" $LIST
```
Comment (TomyLobo) [inline generate-puppetfile]: Why does it say /1/ here instead of /\1/, btw? Keeping it that way for now, maybe you had a reason :)

Comment (rnelson0) [inline generate-puppetfile]: Yes, because $typos! Please go ahead and fix that one.

**Comment by rnelson0** [comment]
@TomyLobo I merged in two other changes from the bugs you pointed out, you'll need to rebase and resolve a conflict on the 1 vs \1 portion.

**Comment by rnelson0** [comment]
@TomyLobo I figured out how to correct the merge conflict and update the branch, so I merged it. The subtle bugs you uncovered are pretty important so I wanted to get it patched. Thanks so much for the work!

**Comment by TomyLobo** [comment]
looks like you got all my changes in

**Pull request closed without merging**